pandas/tests/tseries/frequencies/test_to_offset.py

```diff
@@ -146,15 +147,6 @@ def test_to_offset_pd_timedelta(kwargs, expected):
     assert result == expected
 
 
-def test_to_offset_pd_timedelta_invalid():
-    # see gh-9064
-    msg = "Invalid frequency: 0 days 00:00:00"
-    td = Timedelta(microseconds=0)
-
-    with pytest.raises(ValueError, match=msg):
```

Comment:
is this duplicative?

Reply:
this no longer raises, is added as a case on L140

Reply:
ahh ok (oh the microseconds), sure